app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../lib/tunestakeoutwrapper.rb'
+
+class SuggestionsController < ApplicationController
+  def index
+  end
+
+  def add_favorite
+    @suggestion = params["suggestion"]
+    @user = current_user.uid
+    favorites = TunesTakeout.add_a_favorite(@user, @suggestion)
+    redirect_to root_path
+    #to do -- pass suggestion and user id to tunes-takeout-api and call the post url
+  end
+
+  def favorite
+    @suggestion = params["suggestion"]
+    @user = current_user.uid
+    response = TunesTakeoutWrapper.favorite(@user, @suggestion)
+    redirect_to root_path
+  end
+
+  def search
+    # this method is sending the request to Charles Api and saving in the param_results
+    if (params[:search])
+      @param_results = TunesTakeout.search(params[:search])
+    else
+      @param_results = TunesTakeout.search("strawberries")
```

Comment:
You don't seem to use `@param_results` in any of your views. Therefore I think these can just be local variables rather than instance. I think that makes your intentions clearer. If I create instance variables in the controller methods it's for the purpose of serving up data to the view.